bower.json

```diff
@@ -3,6 +3,7 @@
   "dependencies": {
     "adcssy": "adfinis-sygroup/adcssy#0.9.1",
     "font-awesome": "^4.7.0",
-    "nouislider": "^8.0.1"
+    "nouislider": "^8.0.1",
+    "corejs-typeahead": "adfinis-forks/typeahead.js#master"
```

Comment:
Nope.. No PR because the functionality of the fork is not a common needed thing. It's very custom to our use case.